cfy_manager/utils/common.py

```diff
@@ -116,20 +116,20 @@ def untar(source,
     return destination
 
 
-def copy(source, destination):
+def ensure_destination_dir_exists(destination):
```

Comment:
nothing in this function is related to a "destination" so might well be called "ensure_dir_exists" (or just "mkdir")